### ISSUE TITLE
Learning rate decay in the paper

### ISSUE THREAD

**Issue by kwea123** [XifengGuo/CapsNet-Keras#44]
They use learning rate decay in the paper as it is part of the Adam optimizer:

> we use the Adam optimizer (Kingma
and Ba [2014]) with its TensorFlow default parameters, including the exponentially decaying learning
rate, to minimize the sum of the margin losses in Eq. 4.

**Comment by XifengGuo** [comment]
@kwea123 Same issue with #9 and #35
